src/java.base/share/classes/jdk/internal/foreign/abi/ppc64/aix/AixPPC64Linker.java

```diff
@@ -64,8 +68,8 @@ protected MethodHandle arrangeDowncall(MethodType inferredMethodType, FunctionDe
     }
 
     @Override
-    protected MemorySegment arrangeUpcall(MethodHandle target, MethodType targetType, FunctionDescriptor function, SegmentScope scope) {
-        return CallArranger.arrangeUpcall(target, targetType, function, scope);
+    public UpcallStubFactory arrangeUpcall(MethodType mt, FunctionDescriptor cDesc) {
+        throw new InternalError("arrangeUpcall is not implemented");
```

Comment:
This should mirror upstream, with the exception thrown from an updated `CallArranger`:
```java
    protected UpcallStubFactory arrangeUpcall(MethodType mt, FunctionDescriptor cDesc) {
        return CallArranger.arrangeUpcall(mt, cDesc);
    }
```
Same comment for `ppc64/sysv/SysVPPC64leLinker.java` and `abi/s390x/sysv/SysVS390xLinker.java`.

Reply:
Added `return CallArranger.arrangeUpcall(mt, cDesc)`, and `CallArranger.arrangeUpcall(MethodType, FunctionDescriptor)` throws `InternalError()`